README.rst

```diff
@@ -0,0 +1,76 @@
+Example: Basic Sphinx project for Read the Docs
+===============================================
+
+This example shows the an integration of a basic Sphinx project with Read the Docs. You're encouraged to view it to get inspiration and copy & paste from the files in the source code. If you are using Read the Docs for the first time, have a look at the official `Read the Docs Tutorial <https://docs.readthedocs.io/en/stable/tutorial/index.html>`_.
+
+📚 `docs/ <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/docs/>`_
+    A basic Sphinx project lives in ``docs/``, it was generated using Sphinx defaults
+⚙️ `.readthedocs.yaml <https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/.readthedocs.yaml>`_
+    Read the Docs Build configuration is stored in ``.readthedocs.yaml`` (configuration-as-code)
```

Comment:
```suggestion
    Read the Docs Build configuration is stored in ``.readthedocs.yaml``
```

Not sure what this content was doing, but it seems very out of context without explanation.

Reply:
Oh okay, I hear that buzzword all the time -- we store "configuration as code" because we don't want some snowflake stuff in a database somewhere :)